packages/stores/types.ts

```diff
@@ -200,7 +200,7 @@ type TCommonStoreError = {
     redirect_to: string;
     should_clear_error_on_click: boolean;
     should_show_refresh: boolean;
-    redirectOnClick: () => void;
+    redirectOnClick: (() => void) | string;
```

Comment:
why are setting the string type to redirectOnClick?

Reply:
@balakrishna-deriv for the variable **error** from the store common-store we pass `false` if we aren't going to redirect upon button click. Also we have a variable **should_redirect** which we check: if it is `true` - render `<ButtonLink`, if it is `false` - render `<Button `

Reply:
let me fix, this is the issue